Sriram/Reg.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Registration page</title>
+  </head>
+  <body>
+    <h2><strong>Register</strong></h2>
+
+    <form id="Form">
+      First name:
+      <input type="text" name="firstname" value="First name" required/>
+      Last name:
+      <input type="text" name="lastname" value="Last name" /><br /><br />
+      Gender:
+      <label for="male">Male</label>
+      <input type="radio" name="gender" value="male" required/> Male
```

Comment:
Here when you write a label, it shows up as text beside your radio button. 
So when you write Male in the label tag, writing Male in input tag is not required.  
The correct way is 
```suggestion
      <input type="radio" name="gender" value="male" required/>
```